core/src/themes/ionic.functions.string.scss

```diff
@@ -85,24 +85,35 @@
 // Add Root Selector
 // --------------------------------------------------------------------------------
 // Adds a root selector using host based on the selector passed
+// $root: The selector to add the root to
+// $addHostSelector: The selector to add to the root
```

Comment:
Why are these the same description? Could we improve this? Maybe we should add examples of how to use `add-root-selector` like:

```
add-root-selector(&, "[dir=rtl]")
add-root-selector(".item", ":dir(rtl)", false)
```

I don't know if the above is accurate but I think it is.

Reply:
They're very similar and can make it hard to distinguish. I'll update them.